e2etests/package.json

```diff
@@ -33,6 +39,7 @@
   },
   "scripts": {
     "lint": "jshint --show-non-errors .",
+    "server_valid": "mocha --reporter mocha-multi-reporters --reporter-options configFile=../mocha-reports.json test/server_valid.js",
```

Comment:
nit: "validation". no need for abbreviations here, it's not like we type it a lot.

Reply:
Ok.,

---
In reply to: [220400272](https://github.com/Azure/azure-iot-sdk-node/pull/378#discussion_r220400272) [](ancestors = 220400272)